Practice/index.html

```diff
@@ -4,68 +4,261 @@
+              No one will notice!
+            </p>
+            <a class="btn p-3 bg-secondary rounded-pill">Google that shit!</a>
+          </div>
+        </div>
+      </div>
+    </section>
 
-    <script
-      src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js"
-      integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN"
-      crossorigin="anonymous"
-    ></script>
+    <!-- A set of four icons -->
+    <section id="fourIcons">
+      <div class="container-fluid p-5">
+        <div class="row">
+          <div class="col-12 d-flex flex-column align-items-center">
+            <h2>So fucking creative</h2>
+            <hr class="px-5 py-1" />
+          </div>
+        </div>
+        <div class="row text-center">
+          <div
+            class="
+              col-lg-3 col-md-6 col-sm-12
+              p-2
+              d-flex
+              flex-column
+              align-items-center
+            "
+          >
+            <i class="fas fa-globe-africa fa-4x text-primary"></i>
+            <h3 class="py-3">This Fucking Template</h3>
+            <p>
+              Guaranteed to use the same fucking template that every other
+              bootstrap website uses, downloaded straight from The Web™
+            </p>
+          </div>
+          <div
+            class="
+              col-lg-3 col-md-6 col-sm-12
+              d-flex
+              flex-column
+              align-items-center
+            "
+          >
+            <i class="far fa-paper-plane fa-4x text-primary"></i>
+            <h3 class="py-3">This Set of Four Icons</h3>
+            <p>
+              Look at this cool set of four icons describing different things
+              about us! We use four, because it's the default.
+            </p>
+          </div>
+          <div
+            class="
+              col-lg-3 col-md-6 col-sm-12
+              d-flex
+              flex-column
+              align-items-center
+            "
+          >
+            <i class="far fa-thumbs-up fa-4x text-primary"></i>
+            <h3 class="py-3">Lots of effort</h3>
+            <p>
+              We even changed some of the icons! We take the extra effort to
+              make our designs truly original.
+            </p>
+          </div>
+          <div
+            class="
+              col-lg-3 col-md-6 col-sm-12
+              d-flex
+              flex-column
+              align-items-center
+            "
+          >
+            <i class="far fa-heart fa-4x text-primary"></i>
+            <h3 class="py-3">Made with Love</h3>
+            <p>
+              Because nothing says hard work and talent like editing a few lines
+              of text.
+            </p>
+          </div>
+        </div>
+      </div>
+    </section>
 
+    <!-- bullshit -->
+    <section id="bullshit">
+      <div class="container-fluid">
+        <div class="row">

```

Comment:
See if you can come back to this exercise to make an overlay occur when you hover over these images.
Consider adding some divs right after the image tags and use positioning.